src/themes/teams/components/Divider/dividerStyles.ts

```diff
@@ -24,7 +24,13 @@ const beforeAndAfter = (size, type, variables): ICSSPseudoElementStyle => ({
 })
 
 const dividerStyles: IComponentPartStylesInput = {
-  root: ({ props, variables }: { props: IDividerProps; variables: any }): ICSSInJSStyle => {
+  root: ({
+    props,
+    variables,
+  }: {
+    props: IDividerPropsWithDefaults
+    variables: any
```

Comment:
I propose stating the props here as any, as we haven't finalize how we want to type the params to the styles.

Reply:
actually, I would rather stay it as it is, as there were already props being introduced here before (if we would like to get rid of them, there should be rather a separate PR that will ensure this for all components). Also, speaking about things that we are not agreed yet about - it was, essentially, about the type of styling function used for each of the component's parts - this is the reason we are using literal types for its arguments now.

So, my vote is rather to be consistent with the approach to types that was there before (where `props` were already typed) and just fix those places where strict null checks may fail. Please, @mnajdova, let me know about your thoughts on this

Reply:
The props we provided here was part of a PR where we later decided that, for now, we should omit this until we conclude how to proceed. I am just afraid that we would end with having the prop interfaces in some styles and not in others which will make the styles definition not consistent and the later refactoring confusing in my opinion. I agree that this PR is dedicated in fixing strict null checks, so my proposal is to leave it as is in this PR, and making sure to see if we can address this in the other typing PRs while the strict null checks will still pass.